S2VX.Game/Play/UserInterface/ScoreProcessor.cs

```diff
@@ -27,6 +28,30 @@ public class ScoreProcessor : CompositeDrawable {
         public S2VXSample Hit { get; private set; }
         public S2VXSample Miss { get; private set; }
 
+        public int PerfectCount { get; private set; }
+        public int EarlyCount { get; private set; }
+        public int LateCount { get; private set; }
+        public int MissCount { get; private set; }
+        public SortedList<double> Scores { get; private set; } = new SortedList<double>();
```

Comment:
Only gonna comment this on newly added code, but is new() possible here?

Reply:
Changed